flask_webapp/requirements.txt

```diff
@@ -5,7 +5,7 @@ chardet==3.0.4
 click==7.1.2
 Flask==1.1.2
 Flask-RESTful==0.3.8
-gattlib==0.20200122
+#gattlib==0.20200122
```

Comment:
still haven't figured out what to do with this... @NerdboyQ do we need this module or can we risk removing it?